modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -46,7 +46,7 @@ const RichText = ({
 		}
 	}, [editingLanguageId, editorRef]);
 
-	const currentValue = value ?? predefinedValue;
+	const currentValue = value || predefinedValue;
```

Comment:
Hey @alinedoleron 

This does not fix the issue. The root cause of the bug is that `value` becomes  the same as `predefinedValue`, after `change` event. Checking if it's `null` or falsy is not enough. After `value` changes to truthy, further changes in `predefinedValue` are ignored. 

Please see solution in https://github.com/liferay-forms/liferay-portal/pull/874

Reply:
Hey @markocikos! 

@alinedoleron is in PTO until friday. I will see this now. Thanks for your review!